app/dashboards/hold_dashboard.rb

```diff
@@ -10,6 +10,10 @@ class HoldDashboard < Administrate::BaseDashboard
   ATTRIBUTE_TYPES = {
     thesis: Field::BelongsTo.with_options(searchable: true, 
                                           searchable_fields: ['title']),
+    users: Field::HasMany.with_options(
+      searchable: true,
+      searchable_fields: ['kerberos_id', 'uid', 'display_name']
```

Comment:
After a brief discussion in Slack, I now see a reason to make `uid` a searchable field - thanks for the discourse there.